scripts/barclamp_mgmt_lib.rb

```diff
@@ -67,6 +67,13 @@ def fatal(msg, log = nil, exit_code = 1)
   exit exit_code
 end
 
+def get_yml_paths_from_rpm(component)
+  rpm = "crowbar-#{component}"
+  get_rpm_file_list(rpm).select do |file|
+    file =~ %r!^#{CROWBAR_PATH}/barclamps/([^/]+).yml$!
```

Comment:
the issue will be here https://github.com/vuntz/crowbar/blob/barclamp-install-fixes/scripts/barclamp_mgmt_lib.rb#L94